user-guide/Troubleshooting/Procedures/Investigating_NATS_Issues.md

```diff
@@ -119,14 +119,16 @@ A few details to pay attention to:
 
 This is an example of **nats-server.config** in a NATS cluster of 3 or more Agents:
 
+##### [Up to DataMiner 10.2.0(CU5)/10.2.7](#tab/NATS3PlusObsolete)
```

Comment:
these tabs all show up in the same block (5 together instead of 3+2) and all non-codeblock text after the first tab is gone.